Automattic/wp-calypso
# Rename Giphy block to Gif.

#### Changes proposed in this Pull Request

Renames "Giphy" block to "GIF". Block name, description, directory name, and all CSS classes also are updated. 

New icon: https://material.io/tools/icons/static/icons/outline-image_search-24px.svg

New description.

#### Testing instructions

- Spin up JN instance: https://jurassic.ninja/create?gutenpack&calypsobranch=fix/giphy-rename-to-gif&branch=try/giphy
- Connect Jetpack.
- Navigate to `Settings->Jetpack Constants` and enable `JETPACK_BETA_BLOCKS`
- Verify the block works exactly as before.
- Verify that the name of the block in picker and sidebar is now "GIF"
- Verify new icon
- Verify change to block description in sidebar
- NOTE: Previously created Giphy blocks will no longer work since the name of the block has changed.

Fixes https://github.com/Automattic/wp-calypso/pull/30383
